src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -5047,6 +5048,79 @@ class MavlinkStreamObstacleDistance : public MavlinkStream
 	}
 };
 
+class MavlinkStreamActuatorOutputStatus : public MavlinkStream
+{
+public:
+	const char *get_name() const
+	{
+		return MavlinkStreamActuatorOutputStatus::get_name_static();
+	}
+
+	static const char *get_name_static()
+	{
+		return "ACTUATOR_OUTPUT_STATUS";
+	}
+
+	static uint16_t get_id_static()
+	{
+		return MAVLINK_MSG_ID_ACTUATOR_OUTPUT_STATUS;
+	}
+
+	uint16_t get_id()
+	{
+		return get_id_static();
+	}
+
+	static MavlinkStream *new_instance(Mavlink *mavlink)
+	{
+		return new MavlinkStreamActuatorOutputStatus(mavlink);
+	}
+
+	unsigned get_size()
+	{
+		return _esc_status_sub->is_published() ? (MAVLINK_MSG_ID_ACTUATOR_OUTPUT_STATUS_LEN + MAVLINK_NUM_NON_PAYLOAD_BYTES) :
+		       0;
+	}
+
+private:
+	uint64_t _esc_status_time;
+
+	MavlinkOrbSubscription *_esc_status_sub;
+
+	/* do not allow top copying this class */
+	MavlinkStreamActuatorOutputStatus(MavlinkStreamActuatorOutputStatus &) = delete;
+	MavlinkStreamActuatorOutputStatus &operator = (const MavlinkStreamActuatorOutputStatus &) = delete;
+
+protected:
+	explicit MavlinkStreamActuatorOutputStatus(Mavlink *mavlink) : MavlinkStream(mavlink),
+		_esc_status_time(0),
+		_esc_status_sub(_mavlink->add_orb_subscription(ORB_ID(esc_status)))
+	{
+	}
+
+	bool send(const hrt_abstime t)
+	{
+		esc_status_s esc_status;
+
+		if (_esc_status_sub->update(&_esc_status_time, &esc_status)) {
+			mavlink_actuator_output_status_t msg = {};
```

Comment:
@hamishwillee Is that message supposed to contain the feedback from the actuators? It's not clear to me from the message spec.

Reply:
IMO [ACTUATOR_OUTPUT_STATUS](https://mavlink.io/en/messages/common.html#ACTUATOR_OUTPUT_STATUS) message should provide feedback from actuators because of we can receive controls with [ACTUATOR_CONTROL_TARGET](https://mavlink.io/en/messages/common.html#ACTUATOR_CONTROL_TARGET) message.

Reply:
That was my understanding as well.
However that means the implementation here is not correct.

Reply:
@irsdkv `ACTUATOR_CONTROL_TARGET` contains the values before the mixing.

Reply:
@bkueng Cool, Assigned you clarification for review: https://github.com/mavlink/mavlink/pull/1252

Reply:
@bkueng so is this PR now correct or not? I'm confused.

Reply:
> @bkueng so is this PR now correct or not? I'm confused.

It's not. I thought I wrote that?

Reply:
@irsdkv have you fixed the issue raised here?

Reply:
@julianoes I'm still confused.
What I should fix?